libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -145,7 +145,7 @@ void AP_Arming::check_failed(const enum AP_Arming::ArmingChecks check, bool repo
         return;
     }
     char taggedfmt[MAVLINK_MSG_STATUSTEXT_FIELD_TEXT_LEN+1];
-    hal.util->snprintf((char*)taggedfmt, sizeof(taggedfmt)-1, "PreArm: %s", fmt);
+    hal.util->snprintf((char*)taggedfmt, sizeof(taggedfmt)-1, "PreArm check(s) failed");
```

Comment:
@AkshayMatternet which failing checks do not populate the alert content? Are there checks that populated the fmt? This change appears to hardcode the alert message rather than make it dynamically populate with the failing prearm check

Reply:
How can we see which prearm checks actually failed?